### PR TITLE
bootstrap being pushed from src to dist, via gulp

### PR DESCRIPTION
**Pull request by jpez16**
added gulp task to get bootstrap to dist

### CODE REVIEW

File: src/views/index.pug [hunk 1]
```diff
@@ -10,6 +10,7 @@ html(lang="en")
     meta(name="author" content="University of Waterloo Software Engineering Class of 2021 <https://github.com/sexxis/website/contributors>")
 
     link(href="static/style.css" rel="stylesheet")
+    link(href="bootstrap/css/bootstrap.min.css" rel="stylesheet")
```
Comment (lpan) [inline src/views/index.pug:13]: @justinpezzack so basically `node_modules/bootstrap/dist/` will be exposed as `/bootstrap`

File: gulpfile.js [hunk 2]
```diff
@@ -22,6 +22,21 @@ gulp.task('serve', ['nodemon', 'sass', 'pug'], function() {
...
+gulp.task('bootstrap-fonts', function() {
+  gulp.src("src/assets/lib/bootstrap/fonts/**.*")
+      .pipe(gulp.dest('dist/bootstrap/fonts'));
+});
+
+gulp.task('bootstrap-js', function() {
+  gulp.src("src/assets/lib/bootstrap/js**.*")
+      .pipe(gulp.dest('dist/bootstrap/js'));
+});
+
```
Comment (lpan) [inline gulpfile.js]: maybe we need to :hocho: these

**Comment by lpan** [comment]
dank mr. goose

**Comment by lpan** [comment]
:shipit:

**Comment by lpan** [comment]
kk gonna force merge this one :stuck_out_tongue: 
cc @cchan

**Comment by cchan** [comment]
lgtm :+1: :D

**Pull request merged**